gcloud/bigtable/row.py

```diff
@@ -227,6 +227,80 @@ def to_pb(self):
         return data_pb2.RowFilter(column_qualifier_regex_filter=self.regex)
 
 
+class ColumnRangeFilter(RowFilter):
+    """A row filter to restrict to a range of columns.
+
+    Both the start and end column can be included or excluded in the range.
+    By default, we include them both, but this can be changed with optional
+    flags.
+
+    :type column_family_id: str
+    :param column_family_id: The column family that contains the columns. Must
+                             be of the form ``[_a-zA-Z0-9][-_.a-zA-Z0-9]*``.
+
+    :type start_column: bytes
+    :param start_column: The start of the range of columns. If no value is
+                         used, it is interpreted as the empty string
+                         (inclusive) by the backend.
+
+    :type end_column: bytes
+    :param end_column: The end of the range of columns. If no value is used, it
+                       is interpreted as the infinite string (exclusive) by the
+                       backend.
+
+    :type inclusive_start: bool
+    :param inclusive_start: Boolean indicating if the start column should be
+                            included in the range (or excluded).
+
+    :type inclusive_end: bool
+    :param inclusive_end: Boolean indicating if the end column should be
+                          included in the range (or excluded).
+    """
+
+    def __init__(self, column_family_id, start_column=None, end_column=None,
+                 inclusive_start=True, inclusive_end=True):
+        self.column_family_id = column_family_id
+        self.start_column = start_column
+        self.end_column = end_column
+        self.inclusive_start = inclusive_start
+        self.inclusive_end = inclusive_end
```

Comment:
Does this suffer from the case I [wondered about in #1309](https://github.com/GoogleCloudPlatform/gcloud-python/pull/1309#discussion_r48191156)?